webapp/src/projects.tsx

```diff
@@ -575,66 +599,47 @@ export class ProjectsDetail extends data.Component<ProjectsDetailProps, Projects
-        const linkHref = (youTubeId && !url) ? `https://youtu.be/${youTubeId}` :
-            ((/^https:\/\//i.test(url)) || (/^\//i.test(url)) ? url : '');
+        else if (cardType == "codeExample" || cardType == "example")
+            clickLabel = lf("Open Example");
+        else if (cardType == "forumUrl")
+            clickLabel = lf("Open in Forum");
+        else if (cardType == "template")
+            clickLabel = lf("New Project");
+        else if (youTubeId)
+            clickLabel = lf("Play Video");
 
         return <div className="ui grid stackable padded">
-            {image ? <div className="imagewrapper"><div className="image" style={{ backgroundImage: `url("${image}")` }} /></div> : undefined}
+            {image && <div className="imagewrapper">
+                <div className="image" style={{ backgroundImage: `url("${image}")` }} />
+            </div>}
             <div className="column twelve wide">
                 <div className="segment">
                     <div className="header"> {name} </div>
-                    {tags ? <div className="ui labels">
+                    {tags && <div className="ui labels">
                         {tags.map(tag => <div className={`ui ${tagColors[tag] || ''} label`}>{pxt.Util.rlf(tag)}
-                        </div>)}</div> : undefined}
+                        </div>)}</div>}
                     {descriptions && descriptions.map((desc, index) => {
                         return <p key={`line${index}`} className="detail">
                                 {desc}
                             </p>
                     })}
                     <div className="actions">
-                        {actions.map(action =>
-                            isLink && linkHref ?
-                                <sui.Link
-                                    key={`action_${action.label}`}
-                                    href={linkHref} target={'_blank'}
-                                    icon={action.icon}
-                                    text={action.label}
-                                    className={`ui button approve ${action.icon ? 'icon right labeled' : ''} ${action.className || ''}`}
-                                    onClick={action.onClick}
-                                    onKeyDown={sui.fireClickOnEnter}
-                                />
-                                : <sui.Button
-                                    key={`action_${action.label}`}
-                                    icon={action.icon}
-                                    text={action.label}
-                                    className={`approve ${action.icon ? 'icon right labeled' : ''} ${action.className || ''}`}
-                                    onClick={action.onClick}
-                                    onKeyDown={sui.fireClickOnEnter} />
-                        )}
-                        {isForum && <sui.Button
+                        <sui.Button
```

Comment:
Needed to switch to just being buttons so that all of them would be focusable in the same way (make it so in keyboard navigation it will immediately focus on the open button you actually want to use). Anchor tags don't support autoFocus, and all of them behaved effectively the same way in the first place.

Also removed the unused array for making more actions here - seems to have been added for extensibility, but it was ignored when the forumLink button was added and would make the focusing issue less consistent / just makes things a bit harder to parse.